config/core/test-configs.yaml

```diff
@@ -2272,6 +2284,7 @@ test_configs:
     test_plans:
       - baseline
       - baseline-nfs
+      - kselftest-alsa
```

Comment:
Please drop this one, there isn't enough board capacity to run anything else than baseline.

Reply:
It did pass the tests though: https://staging.kernelci.org/test/plan/id/6259359ece93c2099905f4fd/